new-log-viewer/package.json

```diff
@@ -29,21 +29,26 @@
     "@babel/preset-env": "^7.24.7",
     "@babel/preset-react": "^7.24.7",
     "@babel/preset-typescript": "^7.24.7",
+    "@eslint/js": "^9.9.1",
```

Comment:
Is there a reason for adding these dependencies? Supposedly the equivalents have been listed as peer dependencies of `eslint-config-yscope` and those are automatically installed if you're using npm@7 or higher.

As an aside, most IDEs has built-in support (or via plugins) to auto lint files on save. Here's a sample config for enabling ESLint auto lint in PyCharm: 
<img width="984" alt="image" src="https://github.com/user-attachments/assets/19608bf9-389f-4233-aa26-7a50b6b5f376">